### DISCUSSION TITLE
Return back from footnotes

### DISCUSSION THREAD

**Discussion by shivaraj-bh** [srid/emanote#470] [Ideas]
Once I click on a footnote and reach the end of page, a button to go back would be nice.

Similar to (screengrab from [here](https://flakular.in/blog/nixify-haskell-project-intro#footnote-label)):
<img width="615" alt="Screenshot 2023-12-06 at 6 22 15 PM" src="https://github.com/srid/emanote/assets/23645788/16962fe8-e743-430d-9fd3-82b27c394e97">

**Comment by srid** [comment]
Yea. See https://github.com/srid/emanote/issues/110#issuecomment-1208344912
